frontend/src/metabase/common/utils/column-groups.ts

```diff
@@ -13,5 +13,8 @@ export function getColumnGroupIcon(
   if (groupInfo.isImplicitlyJoinable) {
     return "connections";
   }
-  return "sum";
+  if (groupInfo.isMainGroup) {
```

Comment:
In practice this should be identical (so there is no way to test this). But just in case.